lib/plugins/aws/metrics/awsMetrics.js

```diff
@@ -117,12 +117,11 @@ class AwsMetrics {
 
     if (metrics && metrics.length > 0) {
       const getDatapointsByLabel = Label =>
-        _.chain(metrics)
-          .flatten()
-          .filter({ Label })
-          .map('Datapoints')
-          .flatten()
-          .value();
+        metrics
+          .flat()
```

Comment:
We need to maintain support for Node.js v6, so unfortunally we cannot rely on native `flat` (in that case we need to stick to `_.flatten`)

Reply:
I'll revert that.

Reply:
If I keep the `_.flatten` method, there will be 2 nested `_.flatten` call? Any suggestion?

```javascript
_.flatten(
  _.flatten(metrics)
    .filter(metric => metric.Label === Label)
    .map(metric => metric.Datapoints)
);
```

Reply:
I think that's ok, as in original, we need to _flat_ twice